### PR TITLE
New package: jmeter-bin-5.1.1

### PR DESCRIPTION
**Pull request by knusbaum**

### CODE REVIEW

File: srcpkgs/jmeter/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'jmeter'
+pkgname=jmeter
```
Comment (daniel-eys) [inline srcpkgs/jmeter/template]: This should be `jmeter-bin` as it's installing binaries and not building from source.

File: srcpkgs/jmeter/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+version=5.1.1
+revision=1
+wrksrc="apache-jmeter-5.1.1"
```
Comment (daniel-eys) [inline srcpkgs/jmeter/template]: Please use `$version` instead of `5.1.1`

Comment (daniel-eys) [reply srcpkgs/jmeter/template]: A java-runtime-environment is missing, so `depends="virtual?java-environment"`, see for example `apache-maven-bin`

**Comment by knusbaum** [comment]
Thanks. I'll get this fixed up.

**Comment by knusbaum** [comment]
@daniel-eys Any issues here?

**Comment by daniel-eys** [comment]
Template looks good to me.
A void member has to decide whether to merge, maybe they haven't looked at it yet.
You could try giving them a little hint in irc about this pr.

**Comment by the-maldridge** [comment]
Generally we do not accept -bin packages unless they are needed to break bootstrapping loops.  Please provide a template for jmeter from source, and then that can be merged.

**Comment by knusbaum** [comment]
Closed in favor of #13444

**Pull request closed without merging**